packages/now-routing-utils/src/index.ts

```diff
@@ -265,16 +291,29 @@ export function getTransformedRoutes({
 
   if (typeof rewrites !== 'undefined') {
     const code = 'invalid_rewrites';
-    const errors = rewrites
+    const errorsRegex = rewrites
       .map(r => checkRegexSyntax(r.source))
       .filter(notEmpty);
-    if (errors.length > 0) {
+    if (errorsRegex.length > 0) {
+      return {
+        routes,
+        error: createNowError(
+          code,
+          'Rewrites `source` contains invalid regex',
+          errorsRegex
+        ),
+      };
```

Comment:
In Next.js I added an [err.sh](https://github.com/zeit/next.js/blob/canary/errors/invalid-route-source.md) that tries to explain the error more, should we link it here too?

Reply:
Good idea. Maybe I should copy to this repo?

Reply:
Yeah might be good to keep them separate in case we need to explain something unique to one or the other 👍

Reply:
@ijjk Done!